### PR TITLE
Github link in /pumpkin command

### PR DESCRIPTION
**Pull request by lucas11222**
Add a text that says Github Repository and if you click it opens the github,
 https://snowiiii.github.io/Pumpkin/

## Testing

https://github.com/user-attachments/assets/60d6400b-e1d5-4f0d-bf9b-1882571729c2
Captura de pantalla 2024-12-04 175405

## Checklist
Things need to be done before this Pull Request can be merged.

- [x] Code is well-formatted and adheres to project style guidelines: `cargo fmt`
- [x] Code does not produce any clippy warnings: `cargo clippy`
- [x] All unit tests pass: `cargo test`
- [?] I added new unit tests, so other people don't accidentally break my code by changing other parts of the codebase. [How?](https://doc.rust-lang.org/book/ch11-01-writing-tests.html)

### CODE REVIEW

**Comment by lucas11222** [comment]
wating for https://github.com/Snowiiii/Pumpkin/pull/370

**Comment by lucas11222** [comment]
im wating that command traker pr finish

**Comment by lucas11222** [comment]
Command traker pr is merged soo now we can merge this one

**Comment by lucas11222** [comment]
WHY IS STILL CONFITING

**Comment by lucas11222** [comment]
fixed

**Comment by Snowiiii** [comment]
There is no pupkin command

**Comment by lucas11222** [comment]
> There is no pupkin command

sorry typo i mean pumpkin

File: .gitignore [hunk 1]
```diff
@@ -74,7 +74,7 @@ gradle-app.setting
 *.zip
 *.tar.gz
 *.rar
-
+configuration.toml
```
Comment (Commandcracker) [inline .gitignore:77]: Remove this configuration.toml is already in there

**Review by Commandcracker — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (lucas11222) [reply .gitignore:77]: oh sorry it was me configurating my server

Comment (lucas11222) [reply .gitignore]: fixed

File: .gitignore [hunk 2]
```diff
@@ -74,7 +74,6 @@ gradle-app.setting
 *.zip
 *.tar.gz
 *.rar
-
```
Comment (Commandcracker) [inline .gitignore:77]: Also remove the new line :D

Comment (lucas11222) [reply .gitignore:77]: oh what that dosent show in my ide?

Comment (lucas11222) [reply .gitignore:77]: > Also remove the new line :D

it was github actualy that dosent exist

File: pumpkin/src/command/commands/cmd_pumpkin.rs [hunk 3]
```diff
@@ -59,6 +59,16 @@ impl CommandExecutor for PumpkinExecutor {
                             "Click to Copy Minecraft Version",
                         )))
                         .color_named(NamedColor::Gold),
+                    )
+                    .add_child(
+                        TextComponent::text("Github Repository")
```
Comment (Commandcracker) [inline pumpkin/src/command/commands/cmd_pumpkin.rs:64]: Github -> GitHub

Maybe also add the doc url: https://snowiiii.github.io/Pumpkin/
like **GitHub**, **Documentation**

File: .gitignore (hunk 2 above)
Comment (Commandcracker) [reply .gitignore:77]: Try to remove the file and then just add it back by download it from Snowiiii master

File: pumpkin/src/command/commands/cmd_pumpkin.rs (hunk 3 above)
Comment (lucas11222) [reply pumpkin/src/command/commands/cmd_pumpkin.rs:65]: ok

Comment (Commandcracker) [reply pumpkin/src/command/commands/cmd_pumpkin.rs:65]: No like as a text you can click

Comment (lucas11222) [reply pumpkin/src/command/commands/cmd_pumpkin.rs:65]: how?

Comment (Commandcracker) [reply pumpkin/src/command/commands/cmd_pumpkin.rs:65]: Another TextComponent

Comment (lucas11222) [reply pumpkin/src/command/commands/cmd_pumpkin.rs:65]: oh i didint understand added

**Comment by Snowiiii** [comment]
I have to say that i don't really like the design of the current Text. It has no space between the main text and also uses a new color (i think 3 colors are too much for this little sentence

**Comment by lucas11222** [comment]
> I have to say that i don't really like the design of the current Text. It has no space between the main text and also uses a new color (i think 3 colors are too much for this little sentence

oh sorry its formating that delets the spaces for the text idk why

**Comment by lucas11222** [comment]
> I have to say that i don't really like the design of the current Text. It has no space between the main text and also uses a new color (i think 3 colors are too much for this little sentence

1: the screenshot is old, i fixed the spaces. the color someone gave me a tip for the text putting it blue

**Comment by Snowiiii** [comment]
> > I have to say that i don't really like the design of the current Text. It has no space between the main text and also uses a new color (i think 3 colors are too much for this little sentence
> 
> 1: the screenshot is old, i fixed the spaces. the color someone gave me a tip for the text putting it blue

image

The hovering is also a bit broken, when you are between the spaces it says to copy the server version otherwise the docs or repo

**Comment by lucas11222** [comment]
> > > I have to say that i don't really like the design of the current Text. It has no space between the main text and also uses a new color (i think 3 colors are too much for this little sentence
> > 
> > 
> > 1: the screenshot is old, i fixed the spaces. the color someone gave me a tip for the text putting it blue
> 
> image
> 
> The hovering is also a bit broken, when you are between the spaces it says to copy the server version otherwise the docs or repo

thats wirld that dosent happen to me

**Comment by lucas11222** [comment]
after a LONG TIME becuse i forgot this pull request i fixed the bug

**Comment by Snowiiii** [comment]
Overall looks okay. Still not be biggest fan of the design but it does its Job.
Thank you @lucas11222 :+1:

**Comment by lucas11222** [comment]
> Overall looks okay. Still not be biggest fan of the design but it does its Job. Thank you @lucas11222 👍

No poblem! yea i suck at design xd

**Pull request merged**